### PR TITLE
[DROOLS-3073] Retrieving and using actual package name for Data Objects.

### PR DESCRIPTION
**Pull request by gitgabrio**
@jomarko @Rikkola @danielezonca 
Scope of this PR is to retrieve and store actual package for data objects.
To check:
when using imported Data Objects, running test should not fail with the message " Impossible to load class " followed by a DO with the wrong package name.

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java [hunk 1]
```diff
@@ -372,7 +372,14 @@ private void addRightPanelMenuItem(final FileMenuBuilder fileMenuBuilder) {
...
             }
-            FactModelTree toSend = new FactModelTree(factName, packageName, simpleProperties);
+            String factPackageName;
+            String fullFactClassName = oracle.getFQCNByFactName(factName);
+            if (fullFactClassName != null &&  fullFactClassName.contains(".")) {
+                factPackageName = fullFactClassName.substring(0, fullFactClassName.lastIndexOf("."));
+            } else {
+                factPackageName = packageName;
+            }
+            FactModelTree toSend = new FactModelTree(factName, factPackageName, simpleProperties);
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/editor/ScenarioSimulationEditorPresenter.java]: we could shorten the code slightly, please notice you have two whitespaces between `&&` and `fullFactClassName.contains(".")`
```java
            String factPackageName = packageName;
            String fullFactClassName = oracle.getFQCNByFactName(factName);
            if (fullFactClassName != null && fullFactClassName.contains(".")) {
                factPackageName = fullFactClassName.substring(0, fullFactClassName.lastIndexOf("."));
            }
```

Also don't forget to unit tests.

Manual check pending.

**Comment by gitgabrio** [comment]
@kkufova 
When "description" is empty, an exception is thrown due to a server-side bug ( https://issues.jboss.org/browse/DROOLS-3085 ).
For the moment being, be sure to fill the "description" field.

**Comment by gitgabrio** [comment]
@kkufova @Rikkola 
Add test

**Comment by kkufova** [comment]
@gitgabrio, thanks, the PR looks good to me.

**Comment by danielezonca** [comment]
@jomarko 
Can you please approve so we can merge? Thanks

**Pull request merged**